app/assets/stylesheets/modules/_selects.styl

```diff
@@ -34,5 +34,6 @@
     right 191px
   select.sorts
     background url('../images/sort.svg') no-repeat 48% white
+    color: transparent
```

Comment:
Dammit Firefox. The issue was before, the selected option was showing up on top of the icon of the button. I observed that in Chrome. I'll try some other tricks